cac-prod.yml

```diff
@@ -232,7 +232,7 @@ jenkins:
           executeInitScriptAsRoot: true
           existingStorageAccountName: "mgmtvmimgstoreprod"
           imageReference:
-            id: "/subscriptions/8999dec3-0104-4a27-94ee-6588559729d1/resourceGroups/cnp-vmimages-prod/providers/Microsoft.Compute/images/cnp-jenkins-agent74-20191202093609"
+            id: "/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783/resourceGroups/cnp-vmimages-sandbox/providers/Microsoft.Compute/images/cnp-jenkins-agent74-20200124155803"
```

Comment:
this doesn't have access to pull from here, and managed images need to be rebuilt in each subscription they are used in which is why we prefer to use the image gallery.

you can either:
1. run the jenkins prod image build
2. change to use a managed image and add the required permission for this SP to pull from the gallery